builtin/game/privileges.lua

```diff
@@ -34,15 +34,15 @@ core.register_privilege("basic_privs", "Can modify 'shout' and 'interact' privil
 core.register_privilege("privs", "Can modify privileges")
 
 core.register_privilege("teleport", {
-	description = "Can use /teleport command",
+	description = "Can teleport using /teleport",
 	give_to_singleplayer = false,
 })
 core.register_privilege("bring", {
 	description = "Can teleport other players",
 	give_to_singleplayer = false,
 })
 core.register_privilege("settime", {
-	description = "Can use /time",
+	description = "Can set the time of day using /time",
```

Comment:
'Can use /time command'
Generally i don't think we need to repeat the details of the chat commands here, just refer to them.

Reply:
But “Can use /time” is wrong. You CAN use /time even without this priv. `/time` alone displays the current time. You just can not set the time.

Reply:
Okay, I see Wuzzy's point on this one. The priv is specific to the extended use of the command, not to the command itself, so it's a good change.

Reply:
Aha, i wasn't aware of that, fine then.